test/Gopkg.toml

```diff
@@ -15,44 +15,48 @@ required = [
 
 [[override]]
   name = "k8s.io/api"
-  version = "kubernetes-1.15.3"
+  version = "kubernetes-1.16.4"
 
 [[override]]
   name = "k8s.io/apiextensions-apiserver"
-  version = "kubernetes-1.15.3"
+  version = "kubernetes-1.16.4"
 
 [[override]]
   name = "k8s.io/apimachinery"
-  version = "kubernetes-1.15.3"
+  version = "kubernetes-1.16.4"
 
 [[override]]
   name = "k8s.io/client-go"
-  version = "kubernetes-1.15.3"
+  version = "kubernetes-1.16.4"
 
 [[override]]
   name = "k8s.io/kube-aggregator"
-  version = "kubernetes-1.15.3"
+  version = "kubernetes-1.16.4"
 
 [[constraint]]
-  branch = "master"
   name = "knative.dev/serving-operator"
+  branch = "release-0.13"
 
 [[constraint]]
-  branch = "master"
   name = "github.com/knative/test-infra"
```

Comment:
Let's also change a `github.com/knative/test-infra` to `knative.dev/test-infra`. I think I can provide a commit for that.

Reply:
sure, should be simple. On it.

Reply:
Pushed.